developer/docs/arvo/generators.md

```diff
@@ -159,20 +167,22 @@ else `c`". It is roughly equivalent to `?:(=(~ a) b c)`.
 
 This is run as follows:
 
-    ~fintud-macrep:dojo> :examples-ping|send ~sampel-sipnym
-    >=
-    ~fintud-macrep:dojo> :examples-ping|send ~sampel-sipnym 'how do you do'
-    >=
+```
+~fintud-macrep:dojo/examples> :ping|send ~sampel-sipnym
+>=
+~fintud-macrep:dojo/examples> :ping|send ~sampel-sipnym 'how do you do'
+>=
+```
 
-Which causes `~sampel-sipnym` to print `[%receiving 'howdy']` and
+This causes `~sampel-sipnym` to print `[%receiving 'howdy']` and
 `[%receiving 'how do you do']`.
 
 **Exercises**:
 
--   Create a generator for `:examples-sum` from
-    [State](/developer/docs/arvo/state) so that you can run `:examples-sum|add 5`
+-   Create a generator for `:sum` from
+    [State](/docs/arvo/state) so that you can run `:sum|add 5`
     to add numbers to it.
 
--   Create a generator for `:examples-click` from [Web
-    Apps](/developer/docs/arvo/web-apps) so that you can run
-    `:examples-click|poke` to poke it.
+-   Create a generator for `:click` from [Web
+    Apps](/docs/arvo/web-apps) so that you can run
```

Comment:
These can stay `/developer/docs/...` for now. Galen and I might change this back, but let's keep things consistent for now.